### ISSUE TITLE
For curl to json conversion, the xml type data seems to be coming in key value pairs

### ISSUE THREAD

**Issue by eknathyadav** [curlconverter/curlconverter#618]
For curl to JSON conversion, the XML-type data seems to be coming in key-value pairs. Instead, it should have come in a single-string format like other converters give.
![Screenshot from 2024-01-05 02-20-18](https://github.com/curlconverter/curlconverter/assets/48616375/ccbb9729-8552-43aa-9543-9805cd53b5cf)
![Screenshot from 2024-01-05 02-27-58](https://github.com/curlconverter/curlconverter/assets/48616375/3319e6c5-32e4-4f21-99b8-ef1538033ec2)

By the way, great library. Impressive.

**Comment by verhovsky** [comment]
maybe we need to add a `"raw_data":` key so that you can distinguish

```sh
curl example.com --json '"string"'
```

and

```sh
curl example.com --json 'string'
```

which are both converted to `"data": "string"` since the latter is invalid JSON.

**Comment by eknathyadav** [comment]
hi @verhovsky 

can you please tell me how much time will it take to get these changes reflected in the npm package? When do you plan to release these changes?

**Comment by verhovsky** [comment]
@eknathyadav we published 4.9.0 to npm just now with the change.
